### PR TITLE
Update role-based-access-control.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Organizational Messages Manager role to the document

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3593

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/role-based-access-control.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ You can assign built-in roles to groups without further configuration. You can't
 - **Application Manager**: Manages mobile and managed applications, can read device information and can view device configuration profiles.
 - **Endpoint Security Manager**: Manages security and compliance features, such as security baselines, device compliance, conditional access, and Microsoft Defender for Endpoint.
 - **Read Only Operator**: Views user, device, enrollment, configuration, and application information. Can't make changes to Intune.
+- **Organizational messages manager**: Organizational Messages Managers can manage organizational messages in Intune console.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/role-based-access-control.md:56]:
```suggestion
- **Organizational Messages Manager**: Manages organizational messages in Intune console.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Smritib17** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**